package.json

```diff
@@ -93,10 +93,9 @@
     "jest-cli": "^22.4.3",
     "jest-environment-node": "^22.4.3",
     "jest-environment-node-debug": "^2.0.0",
-    "jest-html-reporter": "^0.5.7",
-    "jest-junit": "^3.6.0",
+    "jest-junit": "5.2.0",
```

Comment:
Vulnerability introduced
See build here: https://wash.zowe.org:8443/blue/organizations/jenkins/brightside/detail/PR-219/2/pipeline

Reply:
Updated version.

Note:

jest-junit will now warn that they deprecated support for `testResultsProcessor`. Unfortunately, we can't move everything into the `results` array due to no support from jest-sonar-reporter. See 3dmind/jest-sonar-reporter#29